src/interfaces/podcast.ts

```diff
@@ -6,5 +6,6 @@ export interface Podcast {
     slug?: string;
     thumbnailUrl: string;
     title: string;
-    url: string;
+    url?: string;
+    _id: string;
```

Comment:
now that we are using UUid do we still need _id ?

Reply:
No I guess we do not

Reply:
then let's remove it.